### DISCUSSION TITLE
What updates are required for domain-specific modules?

### DISCUSSION THREAD

**Discussion by Zhongkai-Wang** [Gilead-Public/gsm#1514] [General]

**Comment by mattroumaya** [comment]
@Zhongkai-Wang - should we move this and #1451, #1452, #1453 to the `gsm.pfs` repo?

**Comment by Zhongkai-Wang** [comment]
> @Zhongkai-Wang - should we move this and #1451, #1452, #1453 to the `gsm.pfs` repo?

@mattroumaya - Wanted to keep an issue for each module in gsm for later integration with the modules. Maybe I can change the subject.

**Comment by mattroumaya** [comment]
@Zhongkai-Wang ah ok that makes sense!

**Comment by jwildfire** [comment]
@Zhongkai-Wang I just converted this from an issue to a discussion since I think there are some details to work out before we're ready to commit any code to gsm. 

Do you have any initial thought on what code changes will be required in {gsm} to support {gsm.endpoints} and other similar domain specific modules?
